app/utils/StringUtils.scala

```diff
@@ -0,0 +1,6 @@
+package utils
+
+object StringUtils {
+  def trimDataEntry(value: String): String =
```

Comment:
As its a generic function, i would call this trimSpaceLineBreaks. Also notices this method removes new line characters from the mid of the string as well, not just start and end, so technically this is not trim but remove, so its better to call it removeSpaceLineBreaks. There was a discussion to remove space in between too as its a common problem with copy paste, so its probably better to do it with a regex

string.replaceAll("[\r\n ]", "")

The list of white space characters is big, see https://qwerty.dev/whitespace/. However, for this ticket we are ok with handling \n \r and punctuation space.